build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1132,6 +1132,7 @@ message ActionResult {
   // [GetActionResultRequest][build.bazel.remote.execution.v2.GetActionResultRequest]
   // message. The server MAY omit inlining, even if requested, and MUST do so if inlining
   // would cause the response to exceed message size limits.
+  // Clients SHOULD NOT populate this field when uploading to the cache.
```

Comment:
SHOULD NOT or MUST NOT?

Reply:
I'm leaning towards SHOULD NOT because it's unlikely that things will break for inline-happy clients and  naïve servers.